src/assets/i18n/cs.json5

```diff
@@ -5115,6 +5115,17 @@
   // "item.matomo-statistics.info.message" : "Click on a data point to summarize by year / month."
   "item.matomo-statistics.info.message" : "Kliknutím na datový bod provedete shrnutí podle roku/měsíce.",
 
+
+  // "item.view.box.author.message": "Author(s):",
+  "item.view.box.author.message": "Autoři",
+
+  // "item.view.box.files.message": ["This item contains", "files"],
+  "item.view.box.files.message": ["Tento záznam obsahuje", "souborů"],
+
+  // "item.view.box.no-files.message": "This item contains no files.",
+  "item.view.box.no-files.message": "Tento záznam neobsahuje soubory.",
```

Comment:
žádné soubory? but in actually czech, not exactly what I wrote.. I just think there should be some form of žádné

Reply:
I copied it from actual production version:
![image](https://github.com/dataquest-dev/dspace-angular/assets/90026355/d16897af-c490-4bfa-bd06-6670c063603b)
![image](https://github.com/dataquest-dev/dspace-angular/assets/90026355/b24f1df1-9177-4d50-9ebe-15d6c0ab6a25)